lhotse/recipes/ljspeech.py

```diff
@@ -0,0 +1,129 @@
+"""
+The LJ Speech Dataset is a public domain speech dataset consisting of 13,100 short audio clips of a single speaker
+reading passages from 7 non-fiction books. A transcription is provided for each clip. Clips vary in length from 1 to
+10 seconds and have a total length of approximately 24 hours.
+
+The texts were published between 1884 and 1964, and are in the public domain. The audio was recorded in 2016-17 by
+the LibriVox project and is also in the public domain.
+
+See https://keithito.com/LJ-Speech-Dataset for more details.
+"""
+
+import logging
+import shutil
+import tarfile
+import urllib.request
+from pathlib import Path
+from typing import Dict, NamedTuple, Optional, Union
+
+import torchaudio
+
+from lhotse.audio import AudioSource, Recording, RecordingSet
+from lhotse.supervision import SupervisionSegment, SupervisionSet
+from lhotse.features import Fbank
+from lhotse.utils import Pathlike
+
+
+def download_and_untar(
+        target_dir: Pathlike = '.',
+        force_download: Optional[bool] = False
+) -> None:
+    target_dir = Path(target_dir)
+    target_dir.mkdir(parents=True, exist_ok=True)
+    dataset_name = 'LJSpeech-1.1'
+    tar_path = target_dir / f'{dataset_name}.tar.bz2'
+    if force_download or not tar_path.is_file():
+        urllib.request.urlretrieve(f'http://data.keithito.com/data/speech/{dataset_name}.tar.bz2', filename=tar_path)
+    corpus_dir = target_dir / dataset_name
+    completed_detector = corpus_dir / '.completed'
+    if not completed_detector.is_file():
+        shutil.rmtree(corpus_dir, ignore_errors=True)
+        with tarfile.open(tar_path) as tar:
+            tar.extractall(path=target_dir)
+            completed_detector.touch()
+
+
+class LJSpeechMetaData(NamedTuple):
+    audio_path: Pathlike
+    audio_info: torchaudio.sox_signalinfo_t
```

Comment:
The torchaudio guys warned us about this, but I think it's okay to merge now and fix all usages in another PR